### PR TITLE
AWS - Case Study

### PR DESCRIPTION
**Pull request by SaraRFerrer**

### CODE REVIEW

File: aws-case-study.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+enables the website to use the resources of multiple servers, therefore resulting in a 
+faster performance. This is a technology that splits
+a physical server into multiple virtual machines that we call "cloud servers". 
+Connecting these is what creates a single network for hosting a website. 
+
+AWS (Amazon Web Services) is a cloud computing platform that is provided by Amazon. AWS includes a mixture
+of infrastructure-as-a-service (IaaS), platform-as-a-service (PaaS) and packaged-software-as-a-service (SaaS). It is also important to note that 
+the concept of cloud hosting was born together with AWS. 
+
+<img width="1024" alt="Skjermbilde 2023-02-08 kl  14 01 55" src="https://user-images.githubusercontent.com/91666428/217537213-e36faff3-28f8-48bd-a1ba-49449574f5ea.png">
```
Comment (Fermain) [inline aws-case-study.md:23]:
```suggestion
![Skjermbilde 2023-02-08 kl  14 01 55](https://user-images.githubusercontent.com/91666428/217537213-e36faff3-28f8-48bd-a1ba-49449574f5ea.png)
```

File: aws-case-study.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+keywords: development platforms, AWS, Cloud Hosting
+tags: Development Platforms, AWS, Cloud Hosting
+sidebar: development-platforms
+permalink: development-platforms/example-product.html
+folder: development-platforms
+---
+
+## Introduction
+
+Amazon Web Services was the first major cloud hosting service. Cloud Hosting is a service that uses
```
Comment (Fermain) [inline aws-case-study.md:12]: There is no need to break lines artificially. Text is wrapped properly when it is rendered. Please remove all of these line breaks mid sentence.

Comment (SaraRFerrer) [reply aws-case-study.md]: The text wrap is fixed. Thank you

**Review by Fermain — COMMENTED** [review]
Hi Sara, this is a good quality introduction to AWS - a huge topic. Thank you for your contribution!

**Pull request merged**